platform/security/common/src/main/java/ddf/security/ws/policy/impl/FilePolicyLoader.java

```diff
@@ -41,7 +41,7 @@
    */
   public FilePolicyLoader(BundleContext context, String policyLocation) throws IOException {
```

Comment:
❓ I believe the only thing that touches this code is its unit test.

@stustison can this file (and its test) be deleted?

Reply:
No, this code is used in downstream projects.